### PR TITLE
Update cryptography examples

### LINKED ISSUE #1879: Update docs to address latest cryptography updates
### Description
`keys.json` files have slightly different valus now:

"memorySize": 2024, "cipher": "aes-256-gcm", are changed

and maybe more needs to be updated, affected pages/guides should be checked via Testnet if still valid.

#### Affected branch
`development`

#### Page location
https://lisk.com/documentation/beta/run-blockchain/become-a-validator.html#how-to-enable-block-generation-for-the-first-time
https://lisk.com/documentation/beta/api/lisk-node-rpc.html
https://lisk.com/documentation/beta/run-blockchain/account-management.html

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1879

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc [hunk 1]
```diff
@@ -387,25 +388,22 @@ That will return the validator keys that are currently imported on the node:
 Without the hash onion, a validator won't be able to receive any rewards for generating new blocks, although the blocks would still be valid in that case.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc]: imho, I would just slightly reword this as follows:
If the hash onion is absent, a validator will be unable to receive rewards for block generation, even though the blocks would remain valid in such a scenario.

**Review by chris529 — APPROVED** [review]
LGTM 👍  I just left one suggestion..

File: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc (hunk 1 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc]:
```suggestion
If the hash onion is absent, a validator will be unable to receive rewards for block generation, even though the blocks would remain valid in such a scenario.
```

**Review by mitsuaki-u — APPROVED** [review]
